weed/storage/store.go

```diff
@@ -41,23 +41,25 @@ type ReadOption struct {
  * A VolumeServer contains one Store
  */
 type Store struct {
-	MasterAddress       pb.ServerAddress
-	grpcDialOption      grpc.DialOption
-	volumeSizeLimit     uint64 // read from the master
-	Ip                  string
-	Port                int
-	GrpcPort            int
-	PublicUrl           string
-	Locations           []*DiskLocation
-	dataCenter          string // optional informaton, overwriting master setting if exists
-	rack                string // optional information, overwriting master setting if exists
-	connected           bool
+	Locations       []*DiskLocation
```

Comment:
No need. Only one instance in memory.

Reply:
> No need. Only one instance in memory.

How to see the top 10 structures in memory?

Reply:
I think that is ordered in terms of bytes saved for each type, not total amount of memory for the server.